libs/core/src/lib/tree/components/tree-item/tree-item.component.html

```diff
@@ -0,0 +1,95 @@
+<div
+    class="fd-tree__item-container"
+    [attr.tabindex]="_containerTabIndex"
+    (click)="_onClick($event)"
+    (keyup.enter)="_onClick($any($event))"
+    (keyup.space)="_onClick($any($event))"
```

Comment:
```suggestion
    (fdkClicked)="_onClick($event)"
```

Reply:
@g-cheishvili fdkClicked does not prevent default, thus, space scrolls the page

Reply:
Could we have fdkClicked catch events and provide options for preventDefault/stopPropagation/stopImmediatePropagation?